### DISCUSSION TITLE
What's happening here in `next.config.js#webpack`

### DISCUSSION THREAD

**Discussion by o-az** [wevm/wagmi#2458] [Q&A]
What's the purpose of this `webpack` config? I'm seeing it in a few templates I visited. Do I need it in my app if I'm using `wagmi`?

https://github.com/wagmi-dev/create-wagmi/blob/74625265f6c987a472f7574977673f5bd629fd1c/templates/next/connectkit/next.config.js#L4-L7

```ts
  webpack: (config) => {
    config.resolve.fallback = { fs: false, net: false, tls: false }
    return config
  },
```

**Comment by jxom** [comment]
It is for WalletConnect v1 compat – they require some Node modules for some reason.
